ibis/impala/client.py

```diff
@@ -296,6 +295,11 @@ def fetchall(self, columnar=False):
         else:
             return self._cursor.fetchall()
 
+    def _get_error_message(self):
+        op = self._cursor._last_operation
+        req = TGetOperationStatusReq(operationHandle=op.handle)
+        resp = op._rpc('GetOperationStatus', req)
+        return resp.errorMessage
```

Comment:
I think it would be preferred to switch out cur.status() so that we're not running an additional query upon error.